drivers/filesystems/fastfat/vfat.h

```diff
@@ -244,6 +240,9 @@ typedef union _DIR_ENTRY DIR_ENTRY, *PDIR_ENTRY;
 #define VCB_IS_SYS_OR_HAS_PAGE  0x0008
 #define VCB_IS_DIRTY            0x4000 /* Volume is dirty */
 #define VCB_CLEAR_DIRTY         0x8000 /* Clean dirty flag at shutdown */
+/* VCB condition state */
+#define VCB_BAD                 0x0000
```

Comment:
I don't see the point of that define. It's not used, and if a volume is not good, what else can it be that bad?